src/partition/InputLines.cc

```diff
@@ -190,7 +221,8 @@ std::vector<Block> const split(fs::path const &path, off_t blockSize) {
 
 class InputLines::Impl {
 public:
-    Impl(std::vector<fs::path> const &paths, size_t blockSize, bool skipFirstLine);
+    Impl(std::vector<fs::path> const &paths, size_t blockSize, bool skipFirstLine,
+         std::vector<std::string> paramNames);
```

Comment:
(for efficiency) Redeclare a type of this input parameter as:
```
std::vector<std::string> const& paramNames = std::vector<std::string>()
```

Reply:
paramNames has been replaced by a struct object that contains more csv formating parameters